migration.py

```diff
@@ -23,21 +24,27 @@
 # NOTE: this is parsed in the main function
 TFE_VCS_CONNECTION_MAP = None
 SENSITIVE_DATA_MAP = None
+SOURCE_WORKSPACE_LIST = []
 
 
-def main(migrator, delete_all, no_confirmation, migrate_all_state, migrate_sensitive_data, tfe_verify_source):
+def main(migrator, delete_all, no_confirmation, migrate_all_state, migrate_sensitive_data, migrate_select_workspaces, tfe_verify_source):
 
     if delete_all:
         migrator.delete_all_from_target(no_confirmation)
     elif migrate_sensitive_data:
         migrator.migrate_sensitive()
     else:
-        migrator.migrate_all(migrate_all_state, tfe_verify_source)
+        migrator.migrate_all(migrate_all_state, migrate_select_workspaces, tfe_verify_source)
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Migrate from one TFE/C org to another TFE/C org')
     parser.add_argument('--vcs-file-path', dest="vcs_file_path", default=DEFAULT_VCS_FILE, \
         help="Path to the VCS JSON file. Defaults to `vcs.json`.")
+    parser.add_argument('--workspace-list-file-path', dest="workspace_list_file_path", \
+        default=DEFAULT_WORKSPACE_LIST_FILE, \
+            help="Path to the Workspace List file. Defaults to `workspace.txt`.")
+    parser.add_argument('--migrate-select-workspaces', dest="migrate_select_workspaces", action="store_true", \
```

Comment:
We will likely want to make the `migrate_all` and `migrate_select` workflows be mutually exclusive and provide feedback to the CLI flags so as not to confuse users.